prdoc/pr_6533.prdoc

```diff
@@ -0,0 +1,15 @@
+title: "Migrate executor into PolkaVM 0.18.0"
+doc:
+  - audience: Runtime Dev
+    description: |
+      Migrate Migrate sc-executor-polkavm into PolkaVM 0.18.0.
+
+crates:
+  - name: pallet-revive
+    bump: minor
```

Comment:
Not touched in this PR.

Reply:
Semantic version check complained about this, i.e. I did not invent it myself. Should I ignore that error?